### DISCUSSION TITLE
Control the active audio device rather than the device selected

### DISCUSSION THREAD

**Discussion by DrewVartanian** [MonitorControl/MonitorControl#1463] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

When using MonitorControl when connected to a monitor and using my laptop as a secondary display, if I am focused on my laptop and adjust the audio, it does not affect the monitor's audio.  I assume that it is trying to adjust the laptop's audio, as that is the focused device, but since the audio is coming out of the monitor, it ends up doing nothing.

### Describe the solution you'd like

MonitorControl should adjust the device's active audio output (the monitor's output in the described situation above), rather than whichever device is focused.

### Describe alternatives you've considered

A setting would also work, but considering the audio can only come from one place, the audio active output should be the only audio that is changed.

### Anything else?

Setup:
M1 Max 2021 Macbook Pro
CalDigit TS4
Asus PG42UQ Display

**Comment by waydabber** [comment, marked as answer]
Hi there,

I already added an option for this:

<img width="842" alt="Screenshot 2023-11-06 at 16 31 55" src="https://github.com/MonitorControl/MonitorControl/assets/37590873/a3a2db8c-333f-4eb7-b0bd-16db37bfa0b6">
